src/caja-pathbar.c

```diff
@@ -719,7 +719,7 @@ caja_path_bar_size_allocate (GtkWidget     *widget,
     GtkRequisition child_requisition;
     GtkAllocation widget_allocation;
 
-    need_sliders = FALSE;
+    need_sliders = TRUE;
```

Comment:
this commit is not 100% like upstream, i nneded to adjust it a bit for get it working.